editions/tw5.com/tiddlers/system/doc-styles.tid

```diff
@@ -286,3 +286,12 @@ ol.doc-github-contributors li {
 	text-overflow: ellipsis;
 }
 
+.doc-op-comparison {
+  table-layout: fixed;
+  width: 80%;
+}
+
+.doc-op-comparison th .doc-operator {
+  background-color: unset;
```

Comment:
we use tab characters for indentation rather than spaces.

Reply:
Corrected with apologies.